languages.toml

```diff
@@ -985,7 +985,8 @@ name = "htmldjango"
 scope = "source.htmldjango"
 injection-regex = "htmldjango"
 language-servers = ["djlsp", "vscode-html-language-server", "superhtml"]
-file-types = []
+file-types = [{ glob = "templates/**/*.html" }]
+roots = ["manage.py"]
```

Comment:
Adding the `roots` config sounds good but I'd like to leave the `file-types` empty. There was some discussion of this on the PRs that added the language. `templates/**/*.html` matches django project structure but I bet it also matches other non-django kinds of templates as well. So to avoid false positives I'd rather have this need explicit configuration in your `~/.config/helix/languages.toml`